packages/lit/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright (c) 2021 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+export * from 'lit-element';
```

Comment:
Seems like `lit-element` should be sufficient here and it should export these others?

Reply:
I did this to hoist those specifiers and shorten the loading waterfall. That way this module doesn't increase the max module depth at all and should be basically no-cost.

Reply:
Did you confirm that there's not a rollup warning when building the bundle due to duplicate names being exported via the `*` (like we had with `_private`)? I'm not sure if rollup is smart enough to know that the duplicated names being exported ultimately resolve to the same symbol.

Reply:
The plan also called for changing the `main` in `lit-element` to point to a new backward-compatible `index.js` which exports all of the stuff the old version did; in which case this line should be `export * from lit-element/lit-element.js` and not the main. Can you do that in this PR? If not, make an issue?